net/wgsd/patches/0001-go.mod-update-coredns-1.12.2.patch

```diff
@@ -0,0 +1,1418 @@
+From ecf1d7e6c17d6cdfd50d7c3f5110e1727d861bc8 Mon Sep 17 00:00:00 2001
+From: Vladimir Ermakov <vooon341@gmail.com>
+Date: Wed, 2 Jul 2025 10:58:51 +0200
+Subject: [PATCH] go.mod: update coredns 1.12.2
+
+Signed-off-by: Vladimir Ermakov <vooon341@gmail.com>
```

Comment:
Do we really need this huge patch?

Reply:
Yes. That actually changes go.mod, which defines version of CoreDNS to use.

The plugin author seems not very active now, but it still serves the purpose well.

Reply:
Possible alternative - make a generic coredns package and make it configurable to include extra plugins, such as wgsd.

But I do not know how to define multiple source urls in the build system.
Note - coredns has build-time plugins, so shipping as a separate package would not work.